src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  background-color: #000;
+}
+
+p,
+div {
+  box-sizing: border-box;
```

Comment:
this property is usually written with `*` selector because it is useful for every element on the page